app/forms/event_disavowal/password_reset_from_disavowal_form.rb

```diff
@@ -31,6 +31,8 @@ def update_user
     end
 
     def mark_profile_inactive
+      return if user.active_profile.blank?
```

Comment:
Nice one-line fix! Do we want to destroy the proofing component if they have an active profile? I thought we never destroyed proofing components.

Reply:
When a profile is minted, the proofing component gets saved to that profile in an immutable kind of form so it will still be there. If a user goes through IdV again then they will have an overwritten proofing_component table so it's probably not as important.